apps-rendering/src/components/editions/byline/index.tsx

```diff
@@ -119,27 +120,33 @@ const styles = (
 };
 
 const largeTextStyles = (
```

Comment:
It looks like this is only called when the arguments are `italic` and `light`, or `normal` and `bold`, so I think it would be cleaner to either have only one argument here.

Reply:
I tried not to change the logic too much, but you're right that it's actually only ever two combinations. I've changed it to a single boolean argument which I agree is a lot cleaner 👍